packages/jsx-email/src/components/button.tsx

```diff
@@ -83,6 +85,20 @@ export const Button: JsxEmailComponent<ButtonProps> = ({
       {children}
     </a>
   );
+  const vmlFallbackLinkContents = (
+    <a
+      rel={rel}
+      href={href}
+      style={{
+        color: textColor,
+        ...baseStyles,
+        ...style
```

Comment:
weird things can happen with `style` here but not sure how one would tease it out otherwise.